### PR TITLE
Reading list

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes ipfs/docs#479.

### CODE REVIEW

**Comment by johnnymatthews** [comment]
Added a summary into the **Concepts** index page. Also removed **External tutorials** page and sidebar item. No sense having them there if there's no plan as to what should go in there right now. Can always put something in there when we have an issue with an idea of what should go on that page.

**Comment by jessicaschilling** [comment]
Feel free to merge as long as @hsanjuan is cool with it. Also, I think you get the prize for the first completed item from the ecosystem audit ...

**Comment by johnnymatthews** [comment]
@jessicaschilling, you can mail my celebratory crate of Guinness to 1 Cold Road, Toronto, Canada.

I'll wait until @hsanjuan is ok with this PR before merging.

File: docs/concepts/further-reading/academic-papers.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## [A practicable approach towards secure key-based routing](https://web.archive.org/web/20170809130252id_/http://www.tm.uka.de/doc/SKademlia_2007.pdf)
+
+**Baumgart, Ingmar and Mies, Sebastian**: _Security is a common problem in completely decentralized peer-to-peer systems. Although several suggestions ex-ist on how to create a secure key-based routing protocol, a practicable approach is still unattended. In this paper we introduce a secure key-based routing protocol based on Kademlia that has a high resilience against common attacks by using parallel lookups over multiple disjoint paths, limiting free nodeId generation with crypto puzzles and intro-ducing a reliable sibling broadcast. The latter is needed tostore data in a safe replicated way. We evaluate the securityof our proposed extensions to the Kademlia protocol ana-lytically and simulate the effects of multiple disjoint pathson lookup success under the influence of adversarial nodes_
+
+## [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/papers/raw/master/ipfs-cap2pfs/ipfs-p2p-file-system.pdf)
```
Comment (hsanjuan) [inline docs/concepts/further-reading/academic-papers.md]: I would put the IPFS paper first on the list of further reading about IPFS :)

Comment (johnnymatthews) [reply docs/concepts/further-reading/academic-papers.md]: Good call.

**Pull request merged**